src/software/thunderscope/robot_communication.py

```diff
@@ -183,7 +183,7 @@ def __enter__(self):
             self.multicast_channel + "%" + self.interface, VISION_PORT, True
         )
 
-        self.connect_fullsystem_to_robots()
+        # self.connect_fullsystem_to_robots()
```

Comment:
uncomment this line and could you add `self.fullsystem_connected_to_robots = True` into the constructor

Reply:
done, this is going to be fixed by #2789 right?

Reply:
Yes, it should be fixed as a part of #2789 so we don't need to change the code to select between AI/diagnostics and which robot to connect to.